src/seedu/addressbook/AddressBook.java

```diff
@@ -517,6 +523,40 @@ private static String getMessageForPersonsDisplayedSummary(ArrayList<String[]> p
 	}
 
 	/**
+	 * Edits the person identified using last displayed index.
+	 *
+	 * @param commandArgs
+	 *            full command args string from the user
+	 * @return feedback display message for the operation result
+	 */
+	
+	private static final String SUCCESSFUL_EDIT_MESSAGE = "Edited successfully!";
+	private static final String FAIL_EDIT_MESSAGE = "Editing failed!";
+	private static String executeEditPerson(String commandArgs) {
+		//decode person from given string
+		final Optional<String[]> decodeResult = decodePersonFromString(commandArgs);
+		int index = getIndexFromInput(commandArgs);
+		String[] personEdits = decodeResult.get();
+		personEdits[0] = removeFirstWord(personEdits[0]);
+		return updateInAddressBook(index, personEdits)?SUCCESSFUL_EDIT_MESSAGE:FAIL_EDIT_MESSAGE;
+	}
+	
+    private static String removeFirstWord(String words){
+    	words = words.substring(words.indexOf(" ")).trim();
+    	return words;
+    }
+	
+    private static int getIndexFromInput(String words){
+    	return Integer.parseInt(words.substring(0, words.indexOf(" ")));
+    }
+
+    private static boolean updateInAddressBook(int index, String[] personEdits){
```

Comment:
Try running the program and editing a value then exit. When you run the program again, list shows the unedited version.